components/base/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react"
+import Paper from "@mui/material/Paper"
+import InputBase from "@mui/material/InputBase"
+import IconButton from "@mui/material/IconButton"
+import SearchIcon from "@mui/icons-material/Search"
+import "./SearchField.scss"
+
+type SearchFieldPropsType = {
+  handleChangeSearch: (value: string | undefined) => void
+  filmName: string | undefined
```

Comment:
комопнент не связан только с фильмами, поэтому названия пропсов тоже не должны упоминать фильмы
```suggestion
  searchValue: string | undefined
```